### PR TITLE
Adding email resolver to mms domain

### PR DESCRIPTION
**Pull request by dabbeg**
## What

Setup a resolver that fetches a teaching license and sends it through an email

## Why

We need this so the user can send a teaching license to some school he is applying for

## Checklist:

- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] Formatting passes locally with my changes
- [x] I have rebased against main before asking for a review

### CODE REVIEW

File: apps/api/src/app/environments/environment.ts [hunk 1]
```diff
@@ -12,6 +12,10 @@ export default {
   },
   education: {
     secret: process.env.EDUCATION_SECRET,
+    emailOptions: {
+      sendFromEmail: 'development@island',
```
Comment (darrikonn) [inline apps/api/src/app/environments/environment.ts:16]: .is

Comment (sindrig) [inline apps/api/src/app/environments/environment.ts:16]: `.is`?

**Review by sindrig — APPROVED** [review]

File: libs/api/domains/education/src/lib/graphql/sendTeachingLicense.input.ts [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import { Allow } from 'class-validator'
```
Comment (darrikonn) [inline libs/api/domains/education/src/lib/graphql/sendTeachingLicense.input.ts:1]: Don't need class validators on graphql types.

File: libs/api/domains/education/src/lib/graphql/main.resolver.ts [hunk 3]
```diff
@@ -20,4 +22,17 @@ export class MainResolver {
   educationTeachingLicense(@CurrentUser() user: User) {
     return this.educationService.getTeachingLicenses(user.nationalId)
   }
+
+  @Mutation(() => SendTeachingLicense, { nullable: true })
+  educationSendTeachingLicense(
```
Comment (darrikonn) [inline libs/api/domains/education/src/lib/graphql/main.resolver.ts:27]: `sendEducationTeachingLicense`
ref: https://github.com/island-is/island.is/blob/main/handbook/technical-overview/api-design-guide/graphql-naming-conventions.md#example

**Review by darrikonn — CHANGES_REQUESTED** [review]

File: libs/api/domains/education/src/lib/graphql/sendTeachingLicense.input.ts (hunk 2 above)
Comment (dabbeg) [reply libs/api/domains/education/src/lib/graphql/sendTeachingLicense.input.ts:1]: I need it, if I skip it I get a validation error

Comment (darrikonn) [reply libs/api/domains/education/src/lib/graphql/sendTeachingLicense.input.ts:1]: WTF

**Review by darrikonn — APPROVED** [review]

**Pull request merged**